Notification popup & Calendar

TharukaN99 opened the pull request:
Added the Notification Popup & the Calendar to date of birth selection in edit profile

TharukaN99 committed:
Notification popup & Calendar

Added the Notification Popup & the Calendar to date of birth selection in edit profile

saminadhikari reviewed (commented):
@TharukaN99 the code looks good for most part. Can you separate `scheduleNotification` function from `main.dart` file to `utilities/notification.dart`. You can create `Notification` class and then add that function there. That way all the future notification logic can be added to that file.

Also if you provide screenshots or recordings to the planner card, that will make it easy to review UI part.

Looks good, just minor refactor part.

TharukaN99 committed:
Updated Notification popup

I separated the notification function from the main.dart file and created a new "nofication.dart"file under the utilities folder.

saminadhikari commented:
Resolving conflicts.

saminadhikari committed:
Merge branch 'main' into main

saminadhikari committed:
Update main.dart
